backend/requirements.prod.txt

```diff
@@ -1,5 +1,6 @@
 gevent==1.5.0
 gunicorn==20.0.4
 psutil==5.7.0
+psycogreen==1.0.2
 psycopg2==2.8.5
 psycopg2==2.8.5
```

Comment:
Was this a weird merge?

Reply:
@Baschdl

Reply:
This one obviously only added `psycogreen==1.0.2` but the duplication of `psycopg` is indedd strange